scalableminds/webknossos
# Remove tree deletion confirm

### URL of deployed dev instance (used for testing):
- https://removetreedeletionconfirm.webknossos.xyz

### Steps to test:
- Open a tracing and delete trees, there should be no confirm and the trees should be deleted.
- Open a task and try to delete the tree that contains the initial node (id 1). There should be a confirm, warning of the deletion.

### Issues:
- fixes #2983

------
